bindings/python/tools/python/setup.cfg.in

```diff
@@ -24,6 +24,7 @@ classifiers =
 zip_safe = True
 include_package_data = True
 install_requires = file: requirements.txt
+packages = find:
```

Comment:
@Derollez during a local install, setuptools was complaining about there being multiple top-level packages: ["ostk", "dist11"] (I had disabled other python versions). This fixes it. Seem okay to you?